.github/workflows/integration.yml

```diff
@@ -30,7 +30,8 @@ env:
   S3_OLAP_ALIAS: ${{ vars.S3_OLAP_ALIAS }}
   S3_OLAP_ARN: ${{ vars.S3_OLAP_ARN }}
   S3_MRAP_ARN: ${{ vars.S3_MRAP_ARN }}
-  RUST_FEATURES: fuse_tests,s3_tests,fips_tests
+  KMS_TEST_KEY_ID: ${{ vars.KMS_TEST_KEY_ID }}
+  RUST_FEATURES: fuse_tests,s3_tests,fips_tests,sse-kms
```

Comment:
Do we actually need the feature flag to be enabled in CI?

I'm hoping we can just hide the CLI arguments behind the feature flag, and leave the rest of the code integrated - just not able to be configured in official releases until its ready.

Reply:
Yeah, if we want to check the emitted logs: https://github.com/awslabs/mountpoint-s3/pull/693/files#diff-b6f1caa405e63c89fd632c2d50612b88726aa868accde4c8c860077ed054403aR562

Reply:
Ohhh I see, gotcha.